stable/magento/README.md

```diff
@@ -59,18 +59,31 @@ The following tables lists the configurable parameters of the Magento chart and
 | `magentoAdminUri`                  | Magento prefix to access Magento Admin   | `admin`                                                  |
 | `mariadb.mariadbRootPassword`      | MariaDB admin password                   | `nil`                                                    |
+| `mariadb.persistence.storageClass` | PVC Storage Class for MariaDB volume     | `nil`  (uses alpha storage annotation)                   |
+| `mariadb.persistence.accessMode`   | PVC Access Mode for MariaDb volume       | `ReadWriteOnce`                                          |
+| `mariadb.persistence.size`         | PVC Storage Request for MariaDB volume   | `8Gi`                                                    |
 | `serviceType`                      | Kubernetes Service type                  | `LoadBalancer`                                           |
 | `persistence.enabled`              | Enable persistence using PVC             | `true`                                                   |
-| `persistence.apache.storageClass`  | PVC Storage Class for Apache volume      | `nil`  (uses alpha storage annotation)                                              |
+| `persistence.apache.storageClass`  | PVC Storage Class for Apache volume      | `nil`  (uses alpha storage annotation)                   |
 | `persistence.apache.accessMode`    | PVC Access Mode for Apache volume        | `ReadWriteOnce`                                          |
 | `persistence.apache.size`          | PVC Storage Request for Apache volume    | `1Gi`                                                    |
-| `persistence.magento.storageClass` | PVC Storage Class for Magento volume     | `nil`  (uses alpha storage annotation)                                                |
+| `persistence.magento.storageClass` | PVC Storage Class for Magento volume     | `nil`  (uses alpha storage annotation)                   |
 | `persistence.magento.accessMode`   | PVC Access Mode for Magento volume       | `ReadWriteOnce`                                          |
 | `persistence.magento.size`         | PVC Storage Request for Magento volume   | `8Gi`                                                    |
 | `resources`                        | CPU/Memory resource requests/limits      | Memory: `512Mi`, CPU: `300m`                             |
 
 The above parameters map to the env variables defined in [bitnami/magento](http://github.com/bitnami/bitnami-docker-magento). For more information please refer to the [bitnami/magento](http://github.com/bitnami/bitnami-docker-magento) image documentation.
 
+A YAML file that specifies the values for the above parameters can be provided while installing the chart. For example,
+
+```console
+$ helm inspect values stable/magento > values.yaml
+$ vi values.yaml
+$ helm install --name my-release -f values.yaml stable/magento
+
+### Installing on GKE
+
+```
 > **Note**:
```

Comment:
I see you have moved this into a Installing on GKE section. But the whole section looks missing/incomplete.